README.md

```diff
@@ -0,0 +1,10 @@
+# Mesh Security SDK
+
+Cosmos module for mesh security. Please see the official project [repo](https://github.com/osmosis-labs/mesh-security)
+for specs and wasm contracts.
+
+## Project Structure
+
+* x - module code
```

Comment:
Maybe say this is the only code that is supposed to be imported by other chains using mesh security.